lua/telescope/pickers.lua

```diff
@@ -986,7 +990,7 @@ end
 
 function Picker:get_result_completor(results_bufnr, prompt, status_updater)
   return function()
-    if self:is_done() then return end
+    if self.closed == true or self:is_done() then return end
```

Comment:
do we need `true` here? I think it could be shortened to `self.closed or self:is_done()`.